cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/autocomplete-config.js

```diff
@@ -22,10 +26,15 @@ function enableAutocomplete(config) {
             },
             placeholder: algoliaData.strings.placeholder,
             getSources(query) {
-                return getSourcesArray(config);
+                return getTrendingItemsArray().then((trendingItems) => {
```

Comment:
It feels weird to have the main sources dependent on the call to `getTrendingItemsArray()`

Isn't it possible to make the recommend call from the `getItems()` function directly? According to the doc, it accepts a promise as return value: https://www.algolia.com/doc/ui-libraries/autocomplete/core-concepts/sources/#param-getitems